.claude/hooks/PreToolUse/invoke_hook_drift_guard.py

```
@@ -0,0 +1,246 @@
+       in the working tree relative to the index; does not mix in
+       committed state.
+    2. ``git ls-files --others --exclude-standard src/copilot-cli/hooks/``:
+       untracked files that the generator may have created.
+
+    A failure to run git is treated as no-diff. Git always works in CI,
+    and a missing git binary at push time is an environmental error that
+    the calling shell will report on its own.
+    """
+    hook_prefix = "src/copilot-cli/hooks/"
+    try:
+        diff_result = subprocess.run(
```

Ruff (0.15.14)

[error] `subprocess` call: check for execution of untrusted input

(S603)